python/podio/frame.py

```diff
@@ -264,7 +264,7 @@ def put_parameter(self, key, value, as_type=None):
                 cpp_types = _get_cpp_types(as_type)
                 if len(cpp_types) == 0:
                     raise ValueError(f"Cannot put a parameter of type {as_type} into a Frame")
-                self._frame.putParameter[cpp_types[0]](key, value)
+                self._frame.putParameter[cpp_types[0][0]](key, value)
```

Comment:
Why is this change necessary? Was this a pre-existing bug?

Reply:
`cpp_types` will be a list of tuples so the first `[0]` gets the first tuple. Before since there were two template arguments (one of them being the EnableIf) it was possible to pass this tuple, now it is not, and the second `[0]` gets the actual C++ type. So not exactly a bug. While having a look at this I also tried to simplify this part a bit but in the end I left it as it is with only a couple of lines in a more pythonic way.